File: LeetCode/C#/2114. Maximum Number of Words Found in Sentences.cs
public class Solution {
    public int MostWordsFound(string[] sentences) {
        var maxWords = 0;
        
        for(var i = 0; i < sentences.Length; i++){
            var words = sentences[i].Split(' ');
            
            maxWords = words.Length > maxWords ? words.Length : maxWords;
        }
        
        return maxWords;
    }
}

File: LeetCode/C++/2114. Maximum Number of Words Found in Sentences.cpp
class Solution
{
public:
    int mostWordsFound(vector<string> &sentences)
    {
        int maxWords = 0;

        for (string s : sentences)
        {
            int wordCount = count(s.begin(), s.end(), ' ') + 1;
            maxWords = max(maxWords, wordCount);
        }

        return maxWords;
    }
};